packages/commons/src/AxeResult.cs

```diff
@@ -3,25 +3,71 @@
 
 namespace Deque.AxeCore.Commons
 {
+    /// <summary>
+    /// Axe results object
+    /// </summary>
     public class AxeResult
     {
+        /// <summary>
+        /// These results indicate what elements failed the rules.
+        /// </summary>
         public AxeResultItem[] Violations { get; }
+
+        /// <summary>
+        /// These results indicate what elements passed the rules.
+        /// </summary>
         public AxeResultItem[] Passes { get; }
+
+        /// <summary>
+        /// These results indicate which rules did not run because no matching content was found on the page. 
+        /// For example, with no video, those rules won't run.
+        /// </summary>
         public AxeResultItem[] Inapplicable { get; }
+
+        /// <summary>
+        /// These results were aborted and require further testing. 
+        /// This can happen either because of technical restrictions to what the rule can test, or because a javascript error occurred.
+        /// </summary>
         public AxeResultItem[] Incomplete { get; }
+
+        /// <summary>
+        /// The date and time that analysis was completed.
+        /// </summary>
         public DateTimeOffset? Timestamp { get; private set; }
+
+        /// <summary>
+        /// Information about the current browser or node application that ran the audit.
+        /// </summary>
         public AxeTestEnvironment TestEnvironment { get; private set; }
+
+        /// <summary>
+        /// The runner that ran the audit.
+        /// </summary>
         public AxeTestRunner TestRunner { get; set; }
 
+        /// <summary>
+        /// The URL of the page that was tested.
+        /// </summary>
         public string Url { get; private set; }
 
+        /// <summary>
+        /// The Error that was found on the application that ran the audit.
```

Comment:
Could not find the equivalent documentation for this property so I made my best guess on what this property is used for.

Reply:
This is something that Selenium.Axe uses to indicate an error that axe-core would normally indicate by throwing. Ideally, we should probably remove this and replace its use with C# exceptions, but that should be a separate PR - what you've done here is fine for this one.